tests/framework/ROM/pickleTests/eRl_setAddtlParams.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2022 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+This test demonstrates the ability for users to change the clusterEvalMode of a ROM with the new
+setAdditionalParams method in the externalROMloader file. This method should allow the user to set any ROM 
+parameters that are present in the pickledROM class; however, that is not yet demonstrated in existing tests.
+The benefit of this method is to allow users to externally evaluate pickled ROM(s) with some ability to 
+customize the evaluation settings.
+'''
+import sys
+import os
+import numpy as np
+
+# Add romLoader to path
+here = os.path.abspath(os.path.dirname(__file__))
+# Paths for the ravenframework and test pk file
+frameworkPath = os.path.abspath(os.path.join(here, *['..']*4, 'ravenframework'))
+picklePath = os.path.join(here,'externalROMloader', 'arma.pk')
+# Appending externalROMloader to path
+sys.path.append(os.path.abspath(os.path.join(frameworkPath, '..', 'scripts')))
+import externalROMloader
+
+def check(runner, before, after, signal):
+    '''
+    Decides of application of setAdditionalParams was successful
+    @ In, runner, externalROMloader object
+    @ In, before/after, list, list of xml objects 
+    @ In, signal, string, name of signal that exists in ROM
+    @ Out, string, 'Pass' or 'Fail'
+    '''
```

Comment:
@AnthoneyGriffith could you follow the raven code standards? This applies to all the methods. This is required for any PR. You can check the link: https://github.com/idaholab/raven/wiki/RAVEN-Software-Commentary-Standard

Reply:
Thank you for pointing those out! I believe I have changed all of them to the correct styling.